letsencrypt/acme/messages2_test.py

```diff
@@ -75,6 +76,12 @@ def test_repr(self):
         self.assertEqual('MockConstant(a)', repr(self.const_a))
         self.assertEqual('MockConstant(b)', repr(self.const_b))
 
+    def test_equality(self):
```

Comment:
Great, thanks! I forgot Python data model for `__eq__/__ne__`. I should probably add `ImmutableMap.__ne__` etc.!